lib/msf/ui/console/command_dispatcher/developer.rb

```diff
@@ -381,25 +387,52 @@ def cmd__servicemanager_help
   # Time how long in seconds a command takes to execute
   #
   def cmd_time(*args)
-    start_time = Process.clock_gettime(Process::CLOCK_MONOTONIC)
-    command = args.join(' ')
-    driver.run_single(command)
-  ensure
-    end_time = Process.clock_gettime(Process::CLOCK_MONOTONIC)
-    elapsed_time = end_time - start_time
-    print_good("Command #{command.inspect} completed in #{elapsed_time} seconds")
+    if args.empty? || args.first == '-h' || args.first == '--help'
+      cmd_time_help
+      return true
+    end
+
+    profiler = nil
+    while args.first == '--cpu' || args.first == '--memory'
+      profiler = args.shift
```

Comment:
Last write wins 🤷

Reply:
Isn't this going to be the first element specified? Aka the opposite of last option specified wins?